packages/hooks/src/use-data-table-sorting-state/use-data-table-sorting-state.js

```diff
@@ -0,0 +1,22 @@
+import { useState, useCallback } from 'react';
+
+const defaultValues = { key: 'createdAt', order: 'desc' };
+
+const useDataTableSortingState = (initialValues = defaultValues) => {
```

Comment:
🤔 do we need this this way? I totally value the explicit merging. I just wonder if `initialValues = defaultValues` is needed as we spread in the `defaultValues` anyway into `mergedValues` and we could spread an `undefined` `initialValues` too?

So my question is would `const useDataTableSortingState = (initialValues) => {` not always be the same?